### PR TITLE
Laravel 10.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l10-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: The latest version of Laravel has a dependency for `phpunit/phpunit` of `^9.5.10`. If you would like to also upgrade your tests, you may run the [PHPUnit 9 Shift](https://laravelshift.com/upgrade-phpunit-9) for free.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/laravel-html-minify.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "dipeshsukhia/laravel-html-minify": "dev-l10-compatibility",
    }
}
```

Finally, run: `composer update`

File: composer.json [hunk 1]
```diff
@@ -17,11 +17,11 @@
...
     "require": {
         "php": ">=5.5.9",
-        "illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|^6.0|^7.0|^8.0|^9.0"
+        "illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|^6.0|^7.0|^8.0|^9.0|^10.0"
     },
     "require-dev": {
-        "orchestra/testbench": "3.8.*",
-        "phpunit/phpunit": "^6.0|^7.0"
+        "orchestra/testbench": "3.8.*|^8.0",
+        "phpunit/phpunit": "^6.0|^7.0|^9.5.10"
```
Comment (pionas) [inline composer.json:24]: in my opinion you should add |^10

**Comment by dipeshsukhia** [comment]
I will check manually for Laravel 10.x Compatibility and testing

**Pull request closed without merging**